File: neetcode/java/src/test/java/solutions/arraysandhashing/ValidAnagramTest.java
package solutions.arraysandhashing;

import static org.junit.Assert.assertTrue;
import org.junit.Test;

public class ValidAnagramTest{
  @Test
  public void example1(){
    String input1 = "anagram";
    String input2 = "nagaram";
    boolean expected = true;
    boolean actual = ValidAnagram.isAnagram(input1, input2);
    assertTrue(expected == actual);
  }

  @Test
  public void example2(){
    String input1 = "rat";
    String input2 = "car";
    boolean expected = false;
    boolean actual = ValidAnagram.isAnagram(input1, input2);
    assertTrue(expected == actual);
  }

  @Test
  public void example3(){
    String input1 = "aacc";
    String input2 = "ccac";
    boolean expected = false;
    boolean actual = ValidAnagram.isAnagram(input1, input2);
    assertTrue(expected == actual);
  }
}

File: neetcode/python/test/arraysandhashing/valid_anagram_test.py
import unittest
from src.arraysandhashing.valid_anagram import Solution

class ValidAnagramTest(unittest.TestCase):
    def test_example1(self):
        arg1 = "anagram"
        arg2 = "nagaram"
        actual = Solution.isAnagram(arg1, arg2)
        expected = True
        self.assertEqual(actual, expected)

    def test_example2(self):
        arg1 = "rat"
        arg2 = "car"
        actual = Solution.isAnagram(arg1, arg2)
        expected = False 
        self.assertEqual(actual, expected)
